plato721/denver-meetings
# Upgrade rails to 5.x and ruby to 2.3.0

- Remove raise_in_transactional_callbacks
- Append rails version (4.2) to all migration files
- Add rails-controller-testing gem to use assigns
- Ensure controller specs use params keyword, not just passing
params hashes directly
- Don't use concat on AR Relations (use .or)
- Also fix the order in here_and_now to order all results by time
- Fix strange and possibly deprecated use of Array#pop in meetings_creator_spec
- Use strong params where they were not being used
- Fix bug caused by misusing `strip!` (I don't think this was related to upgrade)
- Fix geocoder custom_reverse
- Change travis.yml to use ruby 2.3.0
